packages/x-components/src/components/suggestions/base-suggestion.vue

```diff
@@ -10,31 +10,30 @@
     <slot v-bind="{ suggestion, queryHTML }">
       <span v-html="queryHTML" :aria-label="suggestion.query" class="x-suggestion__query" />
     </slot>
-  </BaseEventButton>
+  </button>
 </template>
 
 <script lang="ts">
   import { Suggestion } from '@empathyco/x-types';
   import Vue from 'vue';
   import { Component, Prop } from 'vue-property-decorator';
+  import { QueryFeature } from '../../types';
+  import { forEach } from '../../utils';
```

Comment:
Try to avoid the import from index file and be more specific.

` import { forEach } from '../../utils/object';`